### PR TITLE
New Meteor Cheat Sheet

### PR DESCRIPTION
**Pull request by v3rse**
https://duck.co/ia/view/meteor_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@v3rse Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=meteor cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/meteor_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=meteor cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/meteor_cheat_sheet_mobile.png)

**Comment by tagawa** [comment]
@v3rse, thanks again for creating the Meteor cheat sheet! You’ve been doing a really good job here and we'd like you to help our community tackle the next big challenge - our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Comment by moollaza** [comment]
@v3rse Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@v3rse this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

File: share/goodie/cheat_sheets/json/meteor.json [hunk 1]
```diff
@@ -0,0 +1,246 @@
+{
+    "id": "meteor_cheat_sheet",
+    "name": "Meteor",
+    "description": "A full-stack JavaScript platform for developing modern web and mobile applications. Meteor includes a key set of technologies for building connected-client reactive applications, a build tool, and a curated set of packages from the Node.js and general JavaScript community.",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/meteor.json]: This should be shorter. Ideally a succinct overview of the content

File: share/goodie/cheat_sheets/json/meteor.json [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    "metadata": {
+        "sourceName": "Meteor Docs",
+        "sourceUrl" : "http://docs.meteor.com/#/full/"
+    },
+
+    "aliases": [
+        "meteorjs", "meteor.js","meteor framework"
+    ],
+
+    "template_type": "language",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/meteor.json]: This should be `reference`

**Comment by v3rse** [comment]
@moollaza Do I commit the changes even though the Pull request has been merged?

**Pull request merged**